bin/live/pycbc_live_collate_triggers

```diff
@@ -0,0 +1,306 @@
+        for day in days
+        for trigger_file in os.listdir(os.path.join(args.trigger_dir, day))
+    ]
+
+if args.trigger_file_method == 'start-num-days':
+    assert args.start_date, 'Please provide a start date.'
+    assert args.num_days, 'Please provide a number of days.'
+    assert args.trigger_dir, 'Please provide a directory containing trigger files.'
+    # Convert dates to datetime to get the days array
+    start_date = datetime.strptime(args.start_date, '%Y-%m-%d').date()
+    num_days = timedelta(days=args.num_days - 1)
+    end_date = start_date + num_days
+
+    # Create the days array
+    days = []
+    delta = timedelta(days=1)
+
+    # Loop through the days and add each date to the list
+    current_date = start_date
+    while current_date <= end_date:
+        days.append(current_date)
+        current_date += delta
+
+    # Convert to strings in a list
+    days = [str(date).replace('-', '_') for date in days]
+    num_days = str(len(days))
+
+    trigger_files = [
+        os.path.join(args.trigger_dir, day, trigger_file)
+        for day in days
+        for trigger_file in os.listdir(os.path.join(args.trigger_dir, day))
+    ]
+
+if args.trigger_file_method == 'gps-start-end-time':
+    assert args.gps_start_time, 'Please provide a GPS start time.'
+    assert args.gps_end_time, 'Please provide a GPS end time.'
+    assert args.trigger_dir, 'Please provide a directory containing trigger files.'
+
+    trigger_files = [
+        l for l in glob.glob(args.trigger_dir + '/*/*-Live-*.hdf', recursive=True)
+        if float(l.split('/')[-1].split("-")[2]) > args.gps_start_time and float(l.split('/')[-1].split("-")[2]) < args.gps_end_time
+    ]
+
+start = timeit.default_timer()
+logging.info(f" {len(trigger_files)} files found")
+
+if args.output_trigger_file_list:
+    logging.info(' Writing list of trigger files to: ' + args.output_trigger_file_list)
+    with open(args.output_trigger_file_list, 'w') as f:
+        for item in trigger_files:
+            f.write("%s\n" % item)
+
+# Record start and end gpstime for the output file
+#  This assumes the trigger files are sorted chronologically
+start_gpstime = float(trigger_files[0].split('/')[-1].split("-")[2])
+end_gpstime = float(trigger_files[0].split('/')[-1].split("-")[2])
+
+###########################
+# COLLATE THE TRIGGER FILES
+###########################
+
+if args.output_file_name:
+    output_file = args.output_dir + args.output_file_name
+else:
+    ifo_string = "".join(args.ifos)
+    print(ifo_string)
+    output_file = args.output_dir + f'{ifo_string}-Live-' + start_gpstime + '-' + end_gpstime + '.hdf'
+
+logging.info(f" Creating a file at: {output_file}")
+with h5py.File(output_file, 'a') as destination:
+    # Create the ifo groups in the trigger file
+    for ifo in args.ifos:
+        if ifo not in destination:
+            destination.create_group(ifo)
+
+    # The user must give the ifos in the same order as the trigger files
```

Comment:
I don't understand this comment?

Reply:
There is probably a nice way to do this but the gist is:
--ifos needs to take 'H1 L1 V1' in the same order that the output files from pycbc have the prefix of: H1L1V1-Live-

Reply:
You can't give L1H1V1 because then it looks for L1H1V1 files but these don't exist!

Reply:
I guess you could do something like: `ifo_permutations = [''.join(p) for p in itertools.permutations(ifos)]` and then check for all of the premutations. Or, the live search outputs a known order of ifos every time it runs so we could just force it